charts/gardener/operator/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
This is only temporary, right? Do you want to add a TODO?

Reply:
Why do you think so? The `gardener-operator` is always the first entity that is deployed into the cluster, similar to the `gardenlet`. Hence, they cannot rely on the standard `NetworkPolicy` concept and have dedicated `NetworkPolicy`s.